src/Orleans/Runtime/GrainReference.cs

```diff
@@ -378,7 +398,7 @@ private void CallClientInvokeCallback(InvokeMethodRequest request)
         }
 
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1031:DoNotCatchGeneralExceptionTypes")]
-        private static void ResponseCallback(Message message, TaskCompletionSource<object> context)
+        private void ResponseCallback(Message message, TaskCompletionSource<object> context)
```

Comment:
Ah, that would explain it, do as you prefer if you want to revert it to keep the change relevant to where it's needed, but I don't care. The reason it caught my attention is because of the undo of the other optimization for ResponseCallbackDelegate.